sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/optimizer/InjectRuntimeFilter.scala

```diff
@@ -29,7 +29,14 @@ import org.apache.spark.sql.internal.SQLConf
 import org.apache.spark.sql.types._
 
 /**
- * Insert a filter on one side of the join if the other side has a selective predicate.
+ * Insert a runtime filter on one side of the join (we call this side the application side) if
+ * the plan of other side satisfy one of the following scenes:
```

Comment:
We can be general here and mention details in the corresponding methods. Here can be
```
Insert a runtime filter on one side of the join (we call this side the application side) if we can extract
a runtime filter from the other side (creation side). A simple case is that the creation side is a
table scan with a selective filter.
```

Reply:
To be precise, `the other side` may not be `creation side`, the creation side may be a sub plan of the other side.

So I adjust this comment.

```
Insert a runtime filter on one side of the join (we call this side the application side)
if we can extract a plan (we call this plan the creation side) from the other side and
construct a runtime filter with the creation side. A simple case is that the creation side
is a table scan with a selective filter.
```